examples/react/graphql-request/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@graphql-tools/executor-http": "^0.1.8",
```

Comment:
Maybe this should be another example since `graphql-request` can be overkill in some cases
 @n1ru4l What do you think?

Reply:
Why overkill? I guess both libraries have different features and the tool executor has even more, since it also support subscriptions. So I guess we could also simply rename this example instead and have a separate graphql-request example (or none at all as we would rather advocate people to use our library).

Reply:
So do you think I should just rename this example and revert the changes there?